assets/components/kube-controller-manager/defaultconfig.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeControllerManagerConfig
+extendedArguments:
+  enable-dynamic-provisioning:
+  - "true"
+  allocate-node-cidrs:
+  - "false"
+  configure-cloud-routes:
+  - "false"
+  use-service-account-credentials:
+  - "true"
+  flex-volume-plugin-dir:
+  - "/etc/kubernetes/kubelet-plugins/volume/exec" # created by machine-config-operator, owned by storage team/hekumar@redhat.com
+  pv-recycler-pod-template-filepath-nfs: # owned by storage team/fbertina@redhat.com
+  - "/etc/kubernetes/static-pod-resources/configmaps/recycler-config/recycler-pod.yaml"
+  pv-recycler-pod-template-filepath-hostpath: # owned by storage team/fbertina@redhat.com
+  - "/etc/kubernetes/static-pod-resources/configmaps/recycler-config/recycler-pod.yaml"
+  leader-elect:
+  - "true"
```

Comment:
should be `false`

Reply:
yes, the rebase process will pull this file from kcm-o. overriding in a separate layer makes the rebase easier and also less error prone.
I also added a unit test to verify that microshift doesn't automatically opt in to configuration changes propagating from opeshift. In such a case, the unit test will fail after rebase and we can be aware of the configuration change and then decide whether to opt in or out